src/BugsnagUnity/Payload/SessionReport.cs

```diff
@@ -44,9 +44,22 @@ public SessionReport(Configuration configuration, Dictionary<string, object> cac
             this.AddToPayload("notifier", cachedReport.Get("notifier"));
             this.AddToPayload("app", cachedReport.Get("app"));
             this.AddToPayload("device", cachedReport.Get("device"));
-            this.AddToPayload("sessions", new [] { cachedReport.Get("session") });
+            this.AddToPayload("sessions", cachedReport.Get("sessions"));
             Id = cachedReport["payloadId"].ToString();
         }
 
+        public Dictionary<string, object> GetSerialisableSessionReport()
+        {
+            var serialisableSessionReport = new Dictionary<string, object>
+                {
+                    { "payloadId", Id },
```

Comment:
Session has `payloadId` but event has `id` - is that intended?

Reply:
changed to id 👍